src/notation/internal/notationmidiinput.h

```diff
@@ -57,7 +57,8 @@ class NotationMidiInput : public INotationMidiInput
     mu::engraving::Score* score() const;
 
     void doProcessEvents();
-    Note* onAddNote(const midi::Event& e);
+    Note* addNoteToScore(const midi::Event& e);
```

Comment:
Is that really necessary to return a value here? Looks like it could be 'void' instead

Reply:
The added note is needed for playing
```
    if (note) {
        notes.push_back(note);
    }
    if (!notes.empty()) {
        playbackController()->playElements(notes);
    }
```